src/api/config/initializers/influxdb_rails.rb

```diff
@@ -15,7 +15,12 @@
   config.time_precision      = CONFIG['influxdb_time_precision']
   config.series_name_for_sql = 'rails.sql'
   config.tags_middleware = lambda do |tags|
-    # set a default location when e.g. using ActiveRecord outside of a request
-    { location: :raw }.merge(tags)
+    # set a default tags
+    result = { beta: false, anonymous: true, interface: :none }.merge(tags)
+    result.reject! { |_, value| value.nil? || value == '' } # workaround for bug in influxdb-rails gem
```

Comment:
[nitpick] `value.nil? || value == ''` = `value.blank?`

[nitpick] maybe linking the bug and add a TODO so that this is fixed as some point? :thinking:

Reply:
> [nitpick] value.nil? || value == '' = value.blank?

No, blank does also drop false but I want to keep false.

> [nitpick] maybe linking the bug and add a TODO so that this is fixed as some point? thinking

I will do as soon as I created the PR :smile: